systemd/00-socket-activation.conf

```diff
@@ -0,0 +1,7 @@
+[Unit]
+After=docker.socket
+Requires=docker.socket
+
+[Service]
+ExecStart=
+ExecStart=/usr/bin/dockerd -H unix://
```

Comment:
Should this read `-H fd://`?

Reply:
Yes, it should 

@seemethere ^^

Reply:
Sorry, I see I made that copy/paste mistake ☺️

Reply:
Changed in the latest push